src/haz3lweb/view/LangDoc.re

```diff
@@ -1902,6 +1915,50 @@ let get_doc =
             LangDocMessages.funapp_exp_coloring_ids,
           );
         };
+      | DeferredAp(x, args) =>
+        let x_id = List.nth(x.ids, 0);
+        let supplied_id = (-1);
+        let deferred_id = (-2);
```

Comment:
Hacky: the coloring ids are decoupled with terms. There might be a case in which no arguments are supplied.